tests/Fixtures/TestBundle/Entity/CompositeRelation.php

```diff
@@ -50,7 +50,7 @@ class CompositeRelation
      */
     public function getId()
     {
-        return sprintf('%s-%s', $this->compositeItem->getId(), $this->compositeLabel->getId());
+        return sprintf('%s;%s', $this->compositeItem->getId(), $this->compositeLabel->getId());
```

Comment:
I asked myself too, but : 
If we remove that : 
```json
{
                  "@id": "\/composite_relations\/compositeItem=1;compositeLabel=1",
                  "@type": "CompositeRelation",
                  "id": null,
                  "value": "somefoobardummy"
              }
```

Where id is null, but when this exist, the ID is correct :
` "id": "1;1"`